spec/index.html

```diff
@@ -1046,19 +1045,19 @@ <h2>RDFS Entailment Regime</h2>
             </tr>
             <tr>
               <th>Inconsistency</th>
-              <td>The scoping graph is graph-equivalent to the active graph even if the active graph is <a data-cite="RDF12-SEMANTICS#RDFSINTERP">RDFS-inconsistent</a>. If the active graph is
-              <a data-cite="RDF12-SEMANTICS#RDFSINTERP">RDFS-inconsistent</a>, an implementation MAY raise a
+              <td>The scoping graph is graph-equivalent to the active graph even if the active graph is <a data-cite="RDF12-CONCEPTS#dfn-inconsistent">RDFS-inconsistent</a>. If the active graph is
+              <a data-cite="RDF12-CONCEPTS#dfn-inconsistent">RDFS-inconsistent</a>, an implementation MAY raise a
               <a data-cite="SPARQL12-PROTOCOL#select-refused">QueryRequestRefused</a> fault or issue a warning and it SHOULD generate such a fault or warning if, in the course of processing, it determines that the data or query is not compatible with the request. In
               the presence of an inconsistency the conditions on solutions still guarantee that answers are finite.</td>
             </tr>
             <tr>
               <th>Query Answers</th>
               <td>
                 <p>Let G be the queried RDF graph, BGP be a basic graph pattern, V(BGP) the set of variables in BGP, B(BGP) the set of blank nodes in BGP, SG the
-                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#glossSkolemization">Skolemization</a> of SG with
+                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#skolemization">Skolemization</a> of SG with
                 respect to a vocabulary disjoint from the vocabulary of SG and BGP. Applying sk to a term t, written sk(t), yields sk(t) if sk is defined for t and t otherwise; applying sk to a BGP,
                 written sk(BGP), replaces each blank node b in BGP for which sk is defined with sk(b). The set rdfsV contains the URI references of the
-                <a data-cite="RDF12-SEMANTICS#RDFSINTERP">RDFS vocabulary</a> and rdfsV-Minus is the set of URI references in rdfsV minus URI references of the form
+                <a data-cite="RDF12-SEMANTICS#RDFS vocabulary">RDFS vocabulary</a> and rdfsV-Minus is the set of URI references in rdfsV minus URI references of the form
```

Comment:
As above, `RDF12-SEMANTICS#RDFS vocabulary` is not a valid URI.

Reply:
#dfn-rdfs-vocabulary, fixed (soon)